TTlibrary/spec/controllers/authors_controller_spec.rb

```diff
@@ -0,0 +1,122 @@
+require "spec_helper"
+require "rails_helper"
+
+describe AuthorsController do
+  describe "GET #index" do
+    it "populates an array of authors" do
+      author = create(:author)
```

Comment:
Please use `let` to declare variables that will be used in other parts of the spec. `author` will be used multiple times so it's a good idea to declare it first so that the code is DRY.

Link with information on `let`: https://www.relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let
You can also use `before` blocks instead of `let`. Information here: https://www.relishapp.com/rspec/rspec-core/v/2-2/docs/hooks/before-and-after-hooks